test/unit.js

```diff
@@ -0,0 +1,235 @@
+var test = require('tape');
+var syncCore = require('../core');
+var reqStub = require('./helpers/requestStub');
+var modelStub = require('./helpers/modelStub');
+var Model = require('ampersand-model');
+
+var sync = syncCore(reqStub());
```

Comment:
Plug the stub in once, it exposes options with which it's been called each time.